src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -24,7 +24,7 @@ namespace Microsoft.AspNetCore.Hosting
     /// </summary>
     public class WebHostBuilder : IWebHostBuilder
     {
-        private readonly IHostingEnvironment _hostingEnvironment;
+        private readonly HostingEnvironment _hostingEnvironment;
```

Comment:
Actually I think this is only used in BuildCommonServices so there's no need to keep this class member.